### ISSUE TITLE
Getting error when using Docker installation method

### ISSUE THREAD

**Issue by gooishin** [openvinotoolkit/openvino_notebooks#1951]
**Describe the bug**
I'm installing notebook using Docker. When run the command "docker build -t openvino_notebooks ." , I encounter the error:

```
Step 13/28 : COPY .ci/test_notebooks.tx[t] /tmp/scripts
COPY failed: no source files were specified
```

 **Expected behavior**
Complete install notebook

**Screenshots**
![Screenshot 2024-04-22 132449](https://github.com/openvinotoolkit/openvino_notebooks/assets/100290970/395d7f9f-f685-43eb-b5af-d272ab37af47)

 **Installation instructions (Please mark the checkbox)**
[x] I followed the installation guide at [https://github.com/openvinotoolkit/openvino_notebooks#-installation-guide to install the notebooks. ](https://github.com/openvinotoolkit/openvino_notebooks/wiki/Docker)

** Environment information **
I didnt manage to install using docker method. Hence the error.
```
$ python3 check_install.py 
Importing Jupyter failed. Please follow the installation instructions in the README in the same directory as this script or at https://github.com/openvinotoolkit/openvino_notebooks
```
 
**Additional context**

**Comment by gooishin** [comment]
The Dockerfile change from 

COPY .ci/test_notebooks.tx[t] /tmp/scripts

To:
COPY .ci/test_notebooks.py /tmp/scripts

This will resolve the issue.

**Comment by eaidova** [comment]
@gooishin, sorry for the inconvenience, could you please check the latest repository state? Updating openvino notebooks for 2024.1 I made relevant changes in docker, so now it should work.

**Comment by Aznie-Intel** [comment]
Hi @gooishin, the issue has been addressed by the Pull Request [#1963](https://github.com/openvinotoolkit/openvino_notebooks/pull/1963/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557L53). Updating to OpenVINO notebooks 2024.1 will solve the error. Therefore, I will proceed to close this issue. Feel free to reopen to ask any questions related to this topic.
